src/v2/Apps/Settings/Routes/EditProfile/UserInformation.tsx

```diff
@@ -92,82 +105,69 @@ export const UserInformation: React.FC<UserInformationProps> = ({
           touched,
           values,
         }) => (
-          <Form onSubmit={handleSubmit}>
-            <Text>Full name</Text>
-            <QuickInput
-              error={errors.name as any}
-              placeholder="Enter your full name"
-              name="name"
-              type="text"
-              value={values.name}
-              onChange={handleChange}
-              onBlur={handleBlur}
-            />
-            <Text>Email</Text>
-            <QuickInput
-              block
-              error={errors.email as any}
-              placeholder="Enter your email address"
-              name="email"
-              type="email"
-              value={values.email}
-              onChange={handleChange}
-              onBlur={handleBlur}
-            />
-            <Text>Mobile number</Text>
-            <QuickInput
-              block
-              placeholder="Enter your mobile phone number"
-              name="phone"
-              type="tel"
-              value={values.phone}
-              onChange={handleChange}
-              onBlur={handleBlur}
-            />
-            {me.paddleNumber && (
-              <>
-                <Text>Bidder number</Text>
-                <QuickInput
-                  block
-                  name="paddleNumber"
-                  value={me.paddleNumber}
-                  readOnly
-                />
-              </>
-            )}
-            <input name="internalID" value={me.internalID} hidden readOnly />
-            {touched.email && values.email !== me.email && (
-              <>
-                <Text>Password</Text>
-                <PasswordInput
-                  autoFocus
-                  block
-                  // @ts-expect-error PLEASE_FIX_ME_STRICT_NULL_CHECK_MIGRATION
-                  error={
-                    !values.password && "Password is required to change email."
-                  }
-                  placeholder="Enter your password"
-                  name="password"
-                  value={values.password}
-                  onChange={handleChange}
-                  onBlur={handleBlur}
-                  required
-                />
-              </>
-            )}
-            {status && !status.success && (
-              <Banner variant="error">{status.error}</Banner>
-            )}
-            <Button
-              type="submit"
-              size="large"
-              loading={isSubmitting}
-              width="100%"
-              variant="secondaryOutline"
-            >
-              Save changes
-            </Button>
-          </Form>
+          <form onSubmit={handleSubmit}>
```

Comment:
Is it ok that `<Form>` is now `<form>`?

Reply:
Yeah, the form doesn't really do anything stylistically.

Reply:
Accidental change, reverted.